plenum/test/node_request/test_propagate/test_node_lacks_finalised_requests.py

```diff
@@ -69,4 +72,9 @@ def test_node_request_propagates(looper, setup, txnPoolNodeSet, client1,
     assert requested_propagate_counts[1] == sent_reqs
 
     faulty_node.nodeIbStasher.reset_delays_and_process_delayeds()
-    ensure_pool_functional(looper, txnPoolNodeSet, wallet1, client1, 4)
+    sdk_ensure_pool_functional(looper,
```

Comment:
could we now replace `ensure_pool_functional` with `sdk_ensure_pool_functional` everywhere and remove `ensure_pool_functional` totally?

Reply:
we cannot for the moment